pkg/apis/build/v1alpha1/image_types.go

```diff
@@ -89,3 +89,5 @@ func (*Image) GetGroupVersionKind() schema.GroupVersionKind {
 func (i *Image) NamespacedName() types.NamespacedName {
 	return types.NamespacedName{Namespace: i.Namespace, Name: i.Name}
 }
+
+const ConditionBuilderReady duckv1alpha1.ConditionType = "Builder Ready"
```

Comment:
Is it common to have a space here? Most conditions I have seen would be BuilderReady